Mathlib/Topology/Partial.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2018 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jeremy Avigad
+
+! This file was ported from Lean 3 source module topology.partial
+! leanprover-community/mathlib commit 4c19a16e4b705bf135cf9a80ac18fcc99c438514
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Topology.ContinuousOn
+import Mathlib.Order.Filter.Partial
+
+/-!
+# Partial functions and topological spaces
+
+In this file we prove properties of `Filter.Ptendsto` etc in topological spaces. We also introduce
+`Pcontinuous`, a version of `Continuous` for partially defined functions.
+-/
+
+
+open Filter
+
+open Topology
+
+variable {α β : Type _} [TopologicalSpace α]
+
+theorem rtendsto_nhds {r : Rel β α} {l : Filter β} {a : α} :
+    Rtendsto r l (𝓝 a) ↔ ∀ s, IsOpen s → a ∈ s → r.core s ∈ l :=
```

Comment:
Should it be called `RTendsto`?

Reply:
It's upstream in `Mathlib.Order.Filter.Partial`

In this file we have `Pcontinuous` which follows the same naming, but should be changed as a whole....

Reply:
Recorded this in #2203